timestring/Date.py

```diff
@@ -186,10 +188,9 @@ def __init__(self, date=None, offset: dict = None, tz: str = None,
                 daytime = date.get('daytime')
                 if daytime:
                     if daytime.find('this time') >= 1:
-                        current_time = datetime.now(tz)
-                        new_date = new_date.replace(hour= current_time.hour,
-                                                    minute=current_time.minute,
-                                                    second=current_time.second)
+                        new_date = new_date.replace(hour= now.hour,
```

Comment:
new_date is copy of now. do we still need this replace?

Reply:
Nope. Thank you!